CONTRIBUTING.md

```diff
@@ -1,61 +1,2 @@
-## Bug Reports
```

Comment:
As @borsboom mentioned, this file should not move, since it has special meaning to Github.